client/components/sidebar.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from 'react';
+
+export const Sidebar = () => {
+
+  const [ courseID, setCourseID ] = useState(10);
+  const [ priceData, setPriceData ] = useState();
+  const [ previewVideoData, setPreviewVideoData ] = useState();
+  const [ sidebarData, setSidebarData ] = useState();
+
+  useEffect(() => {
```

Comment:
React Hooks look intriguing. I may try them out

Reply:
I recommend it. Seems like the future, too (though whether we should be learning for the future or not is a matter upon which reasonable minds might differ).